c-pallets/file-bank/src/types.rs

```diff
@@ -59,6 +61,11 @@ pub struct FillerInfo<T: pallet::Config> {
 #[scale_info(skip_type_params(T))]
 #[codec(mel_bound())]
 pub struct UserFileSliceInfo<T: pallet::Config> {
+	// Not sure what kind of hashing algorithm you are using,
+	// but probably better define a new Hash type or reuse the one in
+	// frame_system.
+	// Hashes have fixed size, having a BoundedVec for it might not be
+	// the best option. Some info here: https://substrate.stackexchange.com/questions/4758/what-is-the-best-practice-for-storing-a-hash
```

Comment:
I define a type for it:
https://github.com/CESSProject/cess/pull/88/files#diff-67eede65f7c6cf23f4d0e40dd31f3423594e202ac27bc1e6125268dfd56620fdR59
defined here:
https://github.com/CESSProject/cess/pull/88/files#diff-1b48106f9cdd51de414e128717cc6b39dc2059cc0f9087f3c456ce63c2e4c6d1R12